var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -14,14 +14,15 @@ class Mvapich2(AutotoolsPackage):
     platforms (x86 (Intel and AMD), ARM and OpenPOWER)"""
 
     homepage = "http://mvapich.cse.ohio-state.edu/userguide/userguide_spack/"
-    url = "http://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich2-2.3.5.tar.gz"
+    url = "http://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich2-2.3.6.tar.gz"
     list_url = "http://mvapich.cse.ohio-state.edu/downloads/"
 
-    maintainers = ['nithintsk', 'harisubramoni']
+    maintainers = ['natshineman', 'harisubramoni']
 
     executables = ['^mpiname$']
 
     # Prefer the latest stable release
+    version('2.3.6', sha256='85fb4bad14e80932c593b96a7cd9902be359fdf4042ce985d80150264e85084d')
```

Comment:
This does not match the expected sha256.  I get `b3a62f2a05407191b856485f99da05f5e769d6381cd63e2fcb83ee98fc46a249`.